e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -292,3 +292,21 @@ export function visitPublicDashboard(id, { params = {} } = {}) {
     },
   );
 }
+
+/**
+ * Returns a matcher function to find text content that is broken up by multiple elements
```

Comment:
I'd also add a comment informing that a copy exists in unit tests, and they both should be in sync.

And similar comment in the other function.